### PR TITLE
[WPF] Refactoring redundant path

### PR DESCRIPTION
**Pull request by Potapy4**
## Summary

So, I refactored the WPF section and removed redundant URLs.

Contributes to #10653

cc @mairaw

### CODE REVIEW

File: docs/framework/wpf/advanced/advanced-text-formatting.md [hunk 1]
```diff
@@ -36,15 +36,15 @@ The Windows Presentation Foundation (WPF) provides a robust set of [!INCLUDE[TLA
   
  Unlike a traditional text [!INCLUDE[TLA#tla_api](../../../../includes/tlasharptla-api-md.md)], the <xref:System.Windows.Media.TextFormatting.TextFormatter> interacts with a text layout client through a set of callback methods. It requires the client to provide these methods in an implementation of the <xref:System.Windows.Media.TextFormatting.TextSource> class. The following diagram illustrates the text layout interaction between the client application and <xref:System.Windows.Media.TextFormatting.TextFormatter>.  
   
- ![Diagram of text layout client and TextFormatter](../../../../docs/framework/wpf/advanced/media/textformatter01.png "TextFormatter01")  
+ ![Diagram of text layout client and TextFormatter](media/textformatter01.png "TextFormatter01")
```
Comment (mairaw) [inline docs/framework/wpf/advanced/advanced-text-formatting.md]: I usually prefer to use the . notation to clearly indicate subfolders

```suggestion
 ![Diagram of text layout client and TextFormatter](./media/textformatter01.png "TextFormatter01")  
```

**Review by mairaw — COMMENTED** [review]
This looks really great @Potapy4! I just left a small comment re. subfolders but it's a nit.

**Comment by Potapy4** [comment]
@mairaw Should I continue refactoring other sections as well?

**Comment by mairaw** [comment]
Can you make this an actual PR and not a draft PR so we can merge this? I think you can continue to refactor other sections too, but it's just better if we start a separate PR for them.

**Review by mairaw — APPROVED** [review]
Thank you so much @Potapy4! Looking forward to seeing the next ones.

**Pull request merged**